assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -47,7 +52,12 @@ function CanvasLayout() {
   const layoutParamsCss = useLayoutParamsCssVars();
 
   return (
-    <Background ref={backgroundRef} style={layoutParamsCss}>
+    <Background
+      ref={backgroundRef}
+      style={layoutParamsCss}
+      role="region"
```

Comment:
If `Background` is a `styled.section`, this `role` attribute will not be necessary.

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

> Developers should always prefer using the correct semantic HTML element, in this case `<section>`, over using ARIA.

Reply:
Still not sure if region/section is the right thing here and if aria labels help. It's nice to reference the "real things" in integration tests though.

Reply:
We should not be applying implied roles. `testing-library`'s `*ByRole` selectors work by implicit role too, so it should work correctly without this.

Reply:
Actually the implicit role of a `<section />` will only be `region`, **iff** the node has an accessible name (in terms of `aria-label` or `aria-labelledby`) per [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).

I think switching to `<section />` makes sense here and many other places.

Reply:
Done